### ISSUE TITLE
Modifying an item in a shared mailbox, in another user's mailbox, in an archive mailbox, or in a public folder.

### ISSUE THREAD

**Issue by MattODW** [OfficeDev/outlook-dev-docs#281]
Is item #6 below correct? It states that it is not possible to manipulate an item in another users mailbox by Outlook add-in.  But is seems that it is possible to manipulate items in another user's mail folder by API here:  https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/user_list_messages 
Quote from text on this page:  

Mailbox items available to add-ins
Outlook Add-ins are available on messages or appointments while composing or reading, but not other item types. Outlook does not activate add-ins if the current message item, in a compose or read form, is one of the following:

1. Protected by Information Rights Management (IRM), in S/MIME format or encrypted in other ways for protection. A digitally signed message is an example since digital signing relies on one of these mechanisms.

2. A delivery report or notification that has the message class IPM.Report.*, including delivery and Non-Delivery Report (NDR) reports, and read, non-read, and delay notifications.

3. A draft (does not have a sender assigned to it), or in the Outlook Drafts folder.

4. A .msg or .eml file which is an attachment to another message.

5. A .msg or .eml file opened from the file system.

6. In a shared mailbox, in another user's mailbox, in an archive mailbox, or in a public folder.

7. Using a custom form.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 38eec72d-c430-1f17-5372-ae34fa59995d
* Version Independent ID: 17e688af-f421-f220-54dd-e5aecccd520b
* Content: [Outlook Add-ins overview](https://docs.microsoft.com/en-us/outlook/add-ins/#mailbox-items-available-to-add-ins)
* Content Source: [docs/add-ins/index.md](https://github.com/OfficeDev/outlook-dev-docs/blob/master/docs/add-ins/index.md)
* Product: **outlook**
* GitHub Login: @jasonjoh
* Microsoft Alias: **jasonjoh**

**Comment by jasonjoh** [comment]
Yes, it's correct. What that doc says is that add-ins **do not activate** when you're viewing an item in a shared mailbox, etc. It's not making any assertion about your ability to manipulate items with Microsoft Graph :)

**Comment by jaknudsen** [comment]
Outlook add-ins are activated in shared mailboxes in Outlook 2016 for Mac.
